src/main.py

```diff
@@ -18,6 +19,15 @@ def create_app():
     # Middlewares configuration
     app.add_middleware(ErrorsHandlingMiddleware)
 
+    # Enable CORS for requests from frontend domains
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origin_regex='https://.*\.cmiml\.net',
```

Comment:
Or it should work only such kind of origins?

Reply:
Example is from [https://fastapi.tiangolo.com/tutorial/cors/#:~:text=%27https%3A//.*%5C.example%5C.org%27](https://fastapi.tiangolo.com/tutorial/cors/#use-corsmiddleware) and it works.
Another thing is that it is more correct to do this as a variable, rather than hardcoding.

Reply:
In that case the regex should be changed, isn't it?

Reply:
@Tom1204 No it is not. The dot is a special char in regexes.

```https://.*\.cmiml.net``` will match next `https://mindlogger.cmimlXnet` where X is any character.
`\.` specifies the dot as a mandatory in that example